src/renderers-svg.js

```diff
@@ -509,20 +508,33 @@
                 }
 
                 _attr(this.path, {
-                    "d": makePath(params.d),
+                    "d": makePolygonPath(params.d),
                     "class": clazz,
                     stroke: params.stroke ? params.stroke : null,
                     fill: params.fill ? params.fill : null,
                     transform: "translate(" + offset[0] + "," + offset[1] + ")"
                 });
             }
         };
-        var makePath = function (d) {
-            return (isNaN(d.cxy.x) || isNaN(d.cxy.y)) ? "" : "M" + d.hxy.x + "," + d.hxy.y +
-                " L" + d.tail[0].x + "," + d.tail[0].y +
-                " L" + d.cxy.x + "," + d.cxy.y +
-                " L" + d.tail[1].x + "," + d.tail[1].y +
-                " L" + d.hxy.x + "," + d.hxy.y;
+        var makePolygonPath = function (d) {
```

Comment:
Here I change how we draw arrows so that the rectangle path creation can use the same code.